llama-index-integrations/embeddings/llama-index-embeddings-xinference/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.embeddings.xinference"
+
+[tool.llamahub.class_authors]
+XinferenceEmbedding = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index embeddings xinference integration"
+exclude = ["**/BUILD"]
+license = "MIT"
+name = "llama-index-embeddings-xinference"
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.1"
```

Comment:
We recently released `llama-index-core` v0.11 where we migrate over to pydantic V2. We also updated all of the integrations to be compatible with pydantic V2.

I think it would be best to update this package to use our latest stuff as well.